src/Validator/Constraints/ImageUploadDimensionValidator.php

```diff
@@ -48,7 +48,7 @@ public function validate($value, Constraint $constraint)
             throw new UnexpectedTypeException($value, MediaInterface::class);
         }
 
-        if (null === $value->getBinaryContent()) {
+        if (null === $value->getBinaryContent() || \is_string($value->getBinaryContent())) {
```

Comment:
Can't we improve this check
```suggestion
        if (!$value->getBinaryContent() instanceof \SplFileInfo) {
```
?

Reply:
Thanks